.github/workflows/main.yml

```diff
@@ -41,6 +43,20 @@ jobs:
       publish-prelease: ${{ !!inputs.publish-prerelease }}
     secrets:
       VAULT_URL: ${{ secrets.VAULT_URL }}
+  # Run after publishing according to docs:
+  # https://docs.pact.io/pact_broker/recording_deployments_and_releases#recording-releases
+  pact-record-release:
+    needs: publish
+    uses: ./.github/workflows/pact-record-release.yaml
+    if: >-
+      vars.ENABLE_PACTFLOW == 'true' &&
+      github.ref_name == 'main' &&
+      !contains(github.event.head_commit.message, 'skip-release') &&
+      github.actor != 'dependabot[bot]' &&
```

Comment:
Probably not needed because we're only merging manually on main, but we can keep it as extra precaution.

Reply:
Yes, I thought it would be fine to have more guardrails in place to be prepared if the repo changed over time :)